Source/Core/Core/PowerPC/PowerPC.cpp

```diff
@@ -546,14 +550,50 @@ void CheckExternalExceptions()
   }
 }
 
-void CheckBreakPoints()
+static u32 BreakPointTrigger(const TBreakPoint& breakpoint)
```

Comment:
Naming: Does it make more sense to call it "Trigger Breakpoint" instead of "Breakpoint Trigger"?